releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,986 @@
+
+    * Remove deprecated ContainerVM support from GCE kube-up.  ([#58247](https://github.com/kubernetes/kubernetes/pull/58247), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated kube-push.sh functionality.  ([#58246](https://github.com/kubernetes/kubernetes/pull/58246), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated container-linux support in gce kube-up.sh.  ([#58098](https://github.com/kubernetes/kubernetes/pull/58098), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated and unmaintained photon-controller kube-up.sh.  ([#58096](https://github.com/kubernetes/kubernetes/pull/58096), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated and unmaintained libvirt-coreos kube-up.sh.  ([#58023](https://github.com/kubernetes/kubernetes/pull/58023), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated and unmaintained windows installer.  ([#58020](https://github.com/kubernetes/kubernetes/pull/58020), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated and unmaintained openstack-heat kube-up.sh.  ([#58021](https://github.com/kubernetes/kubernetes/pull/58021), [@mikedanese](https://github.com/mikedanese)
+
+    * Remove deprecated vagrant kube-up.sh. ([#58118](https://github.com/kubernetes/kubernetes/pull/58118),[@roberthbailey](https://github.com/roberthbailey))
+
+* The DaemonSet controller, its integration tests, and its e2e tests, have been updated to use the apps/v1 API. Users should, but are not yet required to, update their scripts accordingly. ([#59883](https://github.com/kubernetes/kubernetes/pull/59883), [@kow3ns](https://github.com/kow3ns)
+
+* MountPropagation feature is now beta. As consequence, all volume mounts in containers are now "rslave" on Linux by default. To make this default work in all Linux environments the entire mount tree should be marked as shareable, e.g. via "mount --make-rshared /". All Linux distributions that use systemd already have root directory mounted as rshared and hence they need not do anything. In Linux environments without systemd we  recommend running "mount --make-rshared /" during boot before docker is started, [@jsafrane](https://github.com/jsafrane)
+
+# Known Issues
+
+This section is still in flux.  Please make sure all issues are documented in the known Issue accumulator: [https://github.com/kubernetes/kubernetes/issues/59764](https://github.com/kubernetes/kubernetes/issues/59764)
+
+* Use of subPath module with hostPath volumes can cause issues during reconstruction ([#61446](https://github.com/kubernetes/kubernetes/issues/61446)) and with containerized kubelets ([#61456](https://github.com/kubernetes/kubernetes/issues/61456)). The workaround for this issue is to specify the complete path in the hostPath volume.
```

Comment:
The workaround for this issue is to not use subPath with hostPath volumes and specify the complete path in the hostPath volume.

Reply:
There are a few more subpath issues.  Should we document those as known issues too? @liggitt @gnufied 
kubernetes/kubernetes#61377: special types of files, such as unix sockets, cannot be mounted via subPath
kubernetes/kubernetes#61545: subpath mounts nested within atomic writer volumes (configmap, secret, downwardAPI, projected) do not work

Reply:
Are there workarounds for these, or are we fixing them later?

Reply:
We also need something like this:
If you need to downgrade from version 1.10 to version 1.9, make sure that you downgrade to version 1.9.6. Otherwise, PersistentVolume and PersistentVolumeClaim objects might not be properly deleted.